eng/pipelines/templates/jobs/archetype-sdk-client.yml

```
@@ -55,7 +69,14 @@ jobs:
         VCPKG_DEFAULT_TRIPLET: 'x64-osx'
         build.args: ' -DBUILD_CURL_TRANSPORT=ON -DAZ_PLATFORM_IMPL=POSIX'
 
-      # Build with sample dependencies and unit testing [curl for transport and cmoka]
+      # Build with sample dependencies and unit testing [curl for transport and cmoka]  - PRECONDITIONS ON
```

nit: typo - cmocka - find/replace in a bunch of places.
```suggestion
      # Build with sample dependencies and unit testing [curl for transport and cmocka]  - PRECONDITIONS ON
```